### ISSUE TITLE
Spartanization

### ISSUE THREAD

**Issue by yossigil** [TechnionYP5777/Info-Evaluation#44]
You were instructed to spartanize you code, but this was not done. I will do this for you.
@yh-github

**Comment by SlasherTech** [comment]
@yossigil 
Do you mean clean up ? (instead of spartanization).
I see that all the changes in our code are only format-wise , and not code-wise.
If that is the case - then I think it stems from different settings of our clean-up routines.
We are using the default clean-up of Eclipse.
Is there any other clean-up routine you would recommend ?

I appreciate your help.

**Comment by yossigil** [comment]
Both. If you check, you will see both spartanization and cleanup.

**Comment by SlasherTech** [comment]
@yossigil 
I saw just now on the webcourse site that there is a new version to the spartanize plug-in.
Therefore , I now see why we had different results when using spartanizer.
I am saying this because some of the code which was changed under your commit didn't change when I was applying the suggested tips from spartanizer . (I always make sure twice that my code is cleaned-up and spartanized before committing).
Hence, we all updated our spartanizer plug-ins to make sure this problem won't happen again.

Thank you.

@geniashand @NetanelFelcher @osherh @ward-mattar @viviansh
